packages/top-app-bar-fixed/README.md

```diff
@@ -0,0 +1,136 @@
+
+```html
+<mwc-top-app-bar-fixed>
+    <mwc-icon-button icon="menu" slot="navigationIcon"></mwc-icon-button>
+    <div slot="title">Title</div>
+    <mwc-icon-button icon="file_download" slot="actionItems"></mwc-icon-button>
+    <mwc-icon-button icon="print" slot="actionItems"></mwc-icon-button>
+    <mwc-icon-button icon="favorite" slot="actionItems"></mwc-icon-button>
+    <div><!-- content --></div>
+</mwc-top-app-bar-fixed>
+```
+
+### Dense
+
+<img src="images/dense.png" height="542px">
+
+```html
+<mwc-top-app-bar-fixed dense>
+    <mwc-icon-button icon="menu" slot="navigationIcon"></mwc-icon-button>
+    <div slot="title">Title</div>
+    <mwc-icon-button icon="file_download" slot="actionItems"></mwc-icon-button>
+    <mwc-icon-button icon="print" slot="actionItems"></mwc-icon-button>
+    <mwc-icon-button icon="favorite" slot="actionItems"></mwc-icon-button>
+    <div><!-- content --></div>
+</mwc-top-app-bar-fixed>
+```
+
+### Prominent
+
+<img src="images/prominent.png" height="542px">
+
+```html
+<mwc-top-app-bar-fixed prominent>
+    <mwc-icon-button icon="menu" slot="navigationIcon"></mwc-icon-button>
+    <div slot="title">Title</div>
+    <mwc-icon-button icon="file_download" slot="actionItems"></mwc-icon-button>
+    <mwc-icon-button icon="print" slot="actionItems"></mwc-icon-button>
+    <mwc-icon-button icon="favorite" slot="actionItems"></mwc-icon-button>
+    <div><!-- content --></div>
+</mwc-top-app-bar-fixed>
+```
+
+### Prominent and Dense
+
+<img src="images/prominent_and_dense.png" height="542px">
+
+```html
+<mwc-top-app-bar-fixed prominent dense>
+    <mwc-icon-button icon="menu" slot="navigationIcon"></mwc-icon-button>
+    <div slot="title">Title</div>
+    <mwc-icon-button icon="file_download" slot="actionItems"></mwc-icon-button>
+    <mwc-icon-button icon="print" slot="actionItems"></mwc-icon-button>
+    <mwc-icon-button icon="favorite" slot="actionItems"></mwc-icon-button>
+    <div><!-- content --></div>
+</mwc-top-app-bar-fixed>
+```
+
+### Customize Colors
+
+<img src="images/custom_colors.png" height="542px">
+
+```css
+mwc-top-app-bar-fixed {
+  --mdc-theme-primary: orange;
+  --mdc-theme-on-primary: black;
+}
+```
+
+## API
+
+### Slots
+| Name | Description
+| ---- | -----------
+| `actionItems` | A number of `<mwc-icon-button>` elements to use for action icons on the right side.
+| `navigationIcon` | `<mwc-icon-button>` elements to use for the left icon.
+| `title` | Element that will be used as the title text.
+| _default_ | Scrollable content to display under the bar. This may be the entire application.
+
+### Properties/Attributes
+| Name | Type | Default | Description
+| ---- | ---- | ------- | -----------
+| `dense` | `boolean` | `false` | Makes the bar a little smaller for higher density applications.
+| `prominent` | `boolean` | `false` | Makes the bar much taller, can be combined with `dense`.
+| `scrollTarget` | `HTMLElement | Window` | `window` | Element used to listen for `scroll` events.
```

Comment:
Missing a closing backtick on `HTMLElement` and opening one for `Window` here, same for the other 2 READMEs.

Reply:
This is the exact typing of `scrollTarget`:  `HTMLElement|Window`. How do we want to handle this for documentation?

Reply:
I think the bar is fine for the union, it's just that it's confusing the markdown parser. Needs an escape or something maybe?

![image](https://user-images.githubusercontent.com/48894/63387665-e2241800-c35a-11e9-96ef-4611d0ca8390.png)

Reply:
Great, `vscode`'s parser works